packages/twenty-front/src/modules/workflow/workflow-diagram/components/WorkflowRunDiagramCanvasEffect.tsx

```diff
@@ -24,27 +24,29 @@ export const WorkflowRunDiagramCanvasEffect = () => {
 
   const workflowId = useRecoilValue(workflowIdState);
 
-  const { activeTabIdState: workflowRunRightDrawerListActiveTabIdState } =
-    useTabListStates({
-      tabListScopeId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
-    });
-
   const goBackToFirstWorkflowRunRightDrawerTabIfNeeded = useRecoilCallback(
     ({ snapshot, set }) =>
       () => {
         const activeWorkflowRunRightDrawerTab = getSnapshotValue(
           snapshot,
-          workflowRunRightDrawerListActiveTabIdState,
+          activeTabIdComponentState.atomFamily({
+            instanceId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
+          }),
         ) as WorkflowRunTabId | null;
 
         if (
           activeWorkflowRunRightDrawerTab === 'input' ||
           activeWorkflowRunRightDrawerTab === 'output'
         ) {
-          set(workflowRunRightDrawerListActiveTabIdState, 'node');
+          set(
+            activeTabIdComponentState.atomFamily({
+              instanceId: WORKFLOW_RUN_STEP_SIDE_PANEL_TAB_LIST_COMPONENT_ID,
+            }),
+            'node',
+          );
         }
```

Comment:
This doesn't seem to work. I expect the tab to never be input or output for workflow run steps.

https://github.com/user-attachments/assets/012d6087-1eff-43bb-9159-f08dafb3769d